web/components/dashboard/widgets/issue-panels/issue-list-item.tsx

```diff
@@ -0,0 +1,297 @@
+import { observer } from "mobx-react-lite";
+import isToday from "date-fns/isToday";
+// hooks
+import { useIssueDetail, useMember, useProject } from "hooks/store";
+// ui
+import { Avatar, AvatarGroup, ControlLink, PriorityIcon } from "@plane/ui";
+// helpers
+import { findTotalDaysInRange, renderFormattedDate } from "helpers/date-time.helper";
+// types
+import { TIssue, TWidgetIssue } from "@plane/types";
+
+type Props = {
+  issueId: string;
+  onClick: (issue: TIssue) => void;
+  workspaceSlug: string;
+};
+
+export const AssignedUpcomingIssueListItem: React.FC<Props> = observer((props) => {
```

Comment:
Please create separate files for each component and re use similar components

Reply:
Each issue list item has different type of data in it, with a different layout.